enclave/server.ts

```diff
@@ -166,6 +203,53 @@ io.on("connection", (socket: Socket) => {
     socket.on("decrypt", (l: Location, pubkey: string, sig: string) => {
         decrypt(socket, l, Player.fromPubString(pubkey), sig);
     });
+
+    nStates.on(nStates.filters.NewMove(), (hUFrom, hUTo) => {
```

Comment:
could we make this a separate callback func 

so instead of an anonymous func with the contents in the listener, we call a function onMoveFinalize() 

also, can you note down that we need to talk about reorgs next time we chat? unsure if this is on finalizing or on confirmation. this is a big deal- need to take this seriously

Reply:
sg. reorgs as in reorganizing the code structue?